scripts/post-update.sh

```diff
@@ -235,10 +237,42 @@ if ! git remote | grep -q local; then
     git remote add local ~/companion
 fi
 
-sudo sed -i '\%stopscreens%d' ~/.bash_aliases
-echo "alias stopscreens=\"screen -ls | grep Detached | cut -d. -f1 | awk '{print \$1}' | xargs kill\"" >> ~/.bash_aliases
+# Check pre-0.0.16 for networking dependencies
+PRE_0_0_16=$(( git rev-list --count --left-right 0.0.16...revert-point || echo 0 ) | cut -f1)
+
+if (( $PRE_0_0_16 > 0 )); then
+    echo "Installing isc-dhcp-server"
+    sudo apt-get update --yes
+    sudo apt-get install isc-dhcp-server=4.3.* --yes
 
-source ~/.bashrc
+    if [ $? -ne 0 ] # If install failed:
+    then
+        echo 'Failed to install isc-dhcp-server; Aborting update'
+        echo 'Rebooting to repair installation, this will take a few minutes'
+        echo 'Please DO NOT REMOVE POWER FROM THE ROV! (until QGC makes a connection again)'
+        sleep 0.1
+        sudo reboot
```

Comment:
will this finish the script ?

Reply:
yes the idea is to abort and revert to pre-update state.

Reply:
There is a lockfile indicating the failure that is checked upon reboot.